src/aiohomeconnect/model/event.py

```diff
@@ -21,14 +22,37 @@ class Event(DataClassJSONMixin):
     """Represent Event."""
 
     key: EventKey
-    name: str | None
-    uri: str | None
     timestamp: int
     level: str
     handling: str
-    value: str | float | bool
-    display_value: str | None = field(metadata=field_options(alias="displayvalue"))
-    unit: str | None
+    value: str | int | float | bool
+    ha_id: str | None = field(default=None, metadata=field_options(alias="haId"))
+    name: str | None = None
+    uri: str | None = None
+    display_value: str | None = field(
+        default=None, metadata=field_options(alias="displayvalue")
+    )
+    unit: str | None = None
+
+
+@dataclass
+class EventMessage:
+    """Represent a server sent event message sent from the Home Connect API."""
+
+    id: str
+    type: EventType
+    data: Event | None
```

Comment:
I don't think `data` should be optional. If the CONNECTED and DISCONNECTED events don't have a `data` attribute we should create a separate model for those events. We should try to avoid optional model attributes as far as possible. If some events are different it's better to use separate models for them. This will allow the library user to just check the event type and then be able use a consistent model for that event type without a lot of checks of optional attributes.

Reply:
I think I've found a simpler way
Now that haId is gone from `Event` class, We can use `ArrayOfEvents` with an empty array at `items` property, that way we use same models for all, and if the items are iterated, the only thing that will happen if no data is sent is that it will iterate over an empty collection (so no iterations).

Also the data for CONNECTED, DISCONNECTED, PAIRED and DEPAIRED, is not really relevant (or at least it would not be relevant in the Home Assistant integration)

Reply:
Yeah, the CONNECTED, DISCONNECTED, PAIRED and DEPAIRED events themselves are highly important!
But the data that contains is not relevant as it provides (if data is provided, you know) information that we already know (i.e. key) or that we don't use right now at the Home Assistant integration (everything else)

I write below the data that contains for reference:
```json
{
  "handling": "none",
  "key": "BSH.Common.Appliance.Disconnected",
  "level": "hint",
  "timestamp": 1733611817,
  "value": true
}
```

Reply:
Yes, I agree. 👍